views/apartments/new.erb

```diff
@@ -1,8 +1,34 @@
-<form method="post" action="/apartments">
+<!-- <form method="post" action="/apartments">
   <input type="text" placeholder="Address">
   <input type="text" placeholder="Monthly Rent">
   <input type="text" placeholder="Square Feet">
   <input type="text" placeholder="Number Beds">
   <input type="text" placeholder="Number Baths">
   <input type="submit">
-</form>
\ No newline at end of file
+</form> -->
+
+<h2>New Apartment</h2>
+
+  <form action="/apartment" method="post">
```

Comment:
See my other comment about forms inside of forms.

When you submit a form, the browser looks inside the form for elements with a `name` attribute, and adds each of those "names" to a `params` hash. The value of each of those keys is the `value` attribute of the element.

For instance:

```
<form>
  <input name="name" value="Bilbo" />
  <input name="age" value="111" />
  <input type="submit" value="Submit" />
</form>
```

This would create a `params` hash like this:

```
{
  "name" => "Bilbo",
  "age" => "111"
}
```